docs/whats_new/version_ongoing.rst

```diff
@@ -52,3 +52,24 @@ The collected messages by the Communication auxiliary can still be cleared with
 :py:meth`~pykiso.lib.auxiliaries.communication_auxiliary.CommunicationAuxiliary.clear_buffer`
 
 See :ref:`communication_auxiliary`
+RTT connector now creates a log folder if it does not exist instead of throwing an error
```

Comment:
There is somehow some duplication here...

Reply:
weird, removed

Reply:
still there

Reply:
Yeah the info is still there but before it was duplicated.